ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -528,36 +517,35 @@ def get_disk_safety(logger):
         Send disk safety for each vpool and the amount of namespaces with the lowest disk safety to DB
         """
 
-        logger.info("Checking if objects need to be repaired...")
-
-        points = []
-
+        logger.info('Checking if objects need to be repaired...')
         test_name = 'disk-safety'
+        abm_services = set(service for service in ServiceHelper.get_services() if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
```

Comment:
Still the same remark, please use meaningful names, these are not ```abm_services```, but just ```services```

Instead do this:
```
abm_services = set(service.abm_service for service in ServiceHelper.get_services() if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
```